app/uk/gov/hmrc/gform/views/form/form.scala.html

```diff
@@ -69,11 +70,18 @@
 @formWithCSRF = @{new formWithCSRF()}
 @govukButton = @{new govukButton()}
 
-@displaySaveContinueButton(classes: String = "") = {
-    @if(shouldDisplayContinue) {
-        @govukButton(Button(content = Text(buttonLabel),
+@displaySaveContinueButton(classes: String = "") = @{
+    val attributes =
+      if(classes === "defaultsink") {
```

Comment:
Wouldn't clearer solution be to pass special attribute as an input?

Something like:
```
displaySaveContinueButton(classes: String = "", otherAttributes: Map[String, String] = Map.empty)  ...
```